int8 OmplSolStatus
string actualmsg
time stamp
